Gemfile

```diff
@@ -11,4 +11,6 @@ gemspec
 # your gem to rubygems.org.
 
 # To use a debugger
-# gem 'byebug', group: [:development, :test]
+gem 'byebug', group: [:development, :test]
```

Comment:
I prefer to use the gemspec file for adding gem dependencies. If they are in this file, the gem will still work as expected, but rubygems (and other gem hosts) won't pick it up as a dependency.

Same goes for below.